src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -4378,6 +4378,74 @@ INSTANTIATE_TEST_SUITE_P(fusings_gpu, conv_before_permute_optimizing, ::testing:
     convolution_test_params{ CASE_CONV_FP16_PERMUTE_2, 3, 2, 4 },
 }));
 
+class EltwiseSumWithConstantFullTensorFusingTestOneDNN : public BaseFusingTest<convolution_eltw_sum_test_params> {
+public:
+    void execute(convolution_eltw_sum_test_params& p) {
+        if (!engine.get_device_info().supports_immad)
+            return;
+        auto input_prim = get_mem(get_weights_layout(p));
+
+        network network_not_fused(this->engine, this->topology_non_fused, cfg_not_fused);
+        network network_fused(this->engine, this->topology_fused, cfg_fused);
+        network_fused.set_input_data("input", input_prim);
+        network_not_fused.set_input_data("input", input_prim);
+
+        for (int i = 0; i < 10; i++) {
+            network_not_fused.execute();
+            network_fused.execute();
+        }
+
+        compare(network_not_fused, network_fused, p);
+
+        auto pi_fused = network_fused.get_primitives_info();
+        auto info_fused = std::find_if(pi_fused.begin(), pi_fused.end(), [](primitive_info& p) -> bool {
+            if (p.original_id == "conv_prim")
+                return true;
+            return false;
+        });
+
+        if (info_fused != pi_fused.end()) {
+            std::cout << "kernel: " << info_fused->kernel_id << std::endl;
+            EXPECT_TRUE(info_fused->kernel_id.find("jit:ir") != std::string::npos);
+        }
```

Comment:
what does this code test? I tried this code(without your patch) and it just succeeds when I increased tolerance..

Reply:
Got it. It will be removed.